drivers/hmis/app/models/hmis/client_alert.rb

```diff
@@ -0,0 +1,10 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::ClientAlert < Hmis::HmisBase
+  belongs_to :created_by, class_name: 'Hmis::Hud::User'
```

Comment:
What is the difference between `Hmis::Hud::User` and `Hmis::User`?

Reply:
Ah, good question. This should actually be a reference to `Hmis::User`, which will be set to the `current_user` that is performing the mutation to create the alert.

* `Hmis::User` and `User` models are both backed by the `users` table in the app database. These are the "application users" aka the user accounts that people log in with.
* `Hmis::Hud::User` is backed by the `User` table in the warehouse database, which matches [User.csv](https://files.hudexchange.info/resources/documents/HMIS-CSV-Format-Specifications-2024.pdf) from HUD. It's really not used for anything except imported data. Its HUD's way of representing users. HUD records, like Client and Enrollment, both store references to that table as specified in the CSV spec.

Unfortunate I know 😞  but the answer is 90% of the time we are working with `Hmis::User`s a.k.a Application Users.